.github/scripts/upload-github.sh

```diff
@@ -0,0 +1,48 @@
+#1/bin/sh
+
+set -eo pipefail
+
+cat > ~/.netrc <<EOF
+machine github.com
+login pytorchbot
+password ${GITHUB_TORCHBENCH_TOKEN}
+EOF
+
+BENCHMARK_DATA_DIR=${HOME}/pytorch_benchmark_data
+BENCHMARK_DATA_REPO=https://github.com/wconstab/pytorch_benchmark_data
```

Comment:
this is just a placeholder, while you add the new repo right?

Reply:
Yes, I will defer this PR until we figure out the new github token